### PR TITLE
adds public process status to view, filter, csv

### PR DESCRIPTION
**Pull request by tillyw**
## Associated issues
fixes https://github.com/cityofaustin/atd-data-tech/issues/10834

this pr adds the field 'public process status' to the project list view (hidden by default), adds an advanced filter with autocomplete, and adds it to the csv export

## Testing
**URL to test:** 

[10834_public_process_status_search_view--atd-moped-main.netlify.app/moped/](https://10834_public_process_status_search_view--atd-moped-main.netlify.app/moped/)

**Steps to test:**
- on the project list view table, enable the new 'public process status' column
- scroll to the right and note the values. if there is no public process status, you should see '-'
- go to a project or multiple and add public process statuses
- go back to the project list view and note the changes
- export the table to csv and note the new column and values
- click the add filter button and select field 'public process status'
- try all the operators with input that should yield results
- selecting the 'is' and 'is not' operators should render dropdowns of all available public process statuses

---
#### Ship list
- [ ] Code reviewed 
- [ ] Product manager approved
- [ ] Added to [QA test script](https://docs.google.com/spreadsheets/d/1n_O6MLh9cwwPf57HUM394Ea-z9uuoEV1-QW4axNZXLE/edit#) if applicable

### CODE REVIEW

File: moped-database/metadata/tables.yaml [hunk 1]
```diff
@@ -4924,30 +4925,31 @@
           - project_sponsor
           - project_tags
           - project_team_members
+          - public_process_status
```
Comment (tillyw) [inline moped-database/metadata/tables.yaml:4928]: this is mostly cleanup except for adding permissions for this field for the different roles

File: moped-database/metadata/tables.yaml [hunk 2]
```diff
@@ -4892,30 +4892,31 @@
...
```
Comment (tillyw) [inline moped-database/metadata/tables.yaml:4895]: this is mostly cleanup except for adding permissions for this field for the different roles

**Review by roseeichelmann — APPROVED** [review]
all the test steps worked perfectly and the code looks good to me! ✨🚢

**Review by johnclary — APPROVED** [review]
looks great + works great ✨ 💯  thank you @tillyw!!!!!


🚢 🚢 🚢 🚢 🚢 🚢 🚢

**Review by mddilley — APPROVED** [review]
I followed all of the test steps, and they all ✅✅✅🙏 This PR is a perfect guide to the config updates for this kind of change. Thanks! 🙌 🚀

**Pull request merged**